docs/content/docs/getting-started.md

```diff
@@ -73,14 +73,14 @@ No problem, you can still add a ``route`` field in the front-matter like this:
 ```md
 ---
 title: My super new page
-route: my-super-url.html
+route: my-super-url
```

Comment:
Looks as though putting an `.html` on the route breaks if that is not the filetype, whereas without the extension it works for both HTML and Markdown pages.